fastplotlib/graphics/_features/_drag.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+
+from . import GraphicFeature, FeatureEvent
+
+
+class DragFeature(GraphicFeature):
+    """
+    Manages mouse and keyboard drag events
+    """
+    def __init__(
+            self,
+            parent,
+            multiplier: np.ndarray = None,
+            clamp: np.ndarray = None,
+    ):
+        """
+        Feature that manages the drag state of a Graphic
+
+        Parameters
+        ----------
+        parent: Graphic
+            parent Graphic that this feature controls
+
+        multiplier: array-like of size 3
+            multiply the drag vector by this multplier vector. final_vector = delta * multiplier.
+            by default the multiplier is set to (1, 1, 1). For example, you can disable movement along an
+            axis by setting the multiplier for that axis component to zero, for example to restrict movement
+            only along the x-axis: use ``multiplier = (1, 0, 0)``
+
+        clamp: np.ndarray, of shape (2, 2) or (2, 3)
+            restrict drag movement of the Graphic within the clamp limits. Argument must be in the form:
+                [[xmin, ymin, zmin],
+                 [xmax, ymax, zmax]]
+
+            by default the clamp is [-inf, +inf] for all axes.
+
+        """
+        super(DragFeature, self).__init__(parent, data=None)
+        self.multiplier = multiplier
+        self.clamp = clamp
+
+        self._last_position = None
+        self._initial_controller_state: bool = None
+
+        self._event_type = None  # event that caused the delta, one of "mouse" or "kb"
+
+    @property
+    def multiplier(self) -> np.ndarray:
+        """
+        multiply the drag vector by this multplier vector. final_vector = delta * multiplier.
+        by default the multiplier is set to (1, 1, 1). For example, you can disable movement along an
+        axis by setting the multiplier for that axis component to zero, for example to restrict movement
+        only along the x-axis: use ``multiplier = (1, 0, 0)``
+        """
+        return self._multiplier
+
+    @multiplier.setter
+    def multiplier(self, value: np.ndarray):
```

Comment:
Should there be more parsing here for size of `value` passed? Should always be a `(1,3)`

Also, should we allow passing a single `float` that would apply to `xyz`?